var/spack/repos/builtin/packages/aml/package.py

```diff
@@ -23,3 +23,38 @@ class Aml(AutotoolsPackage):
     depends_on('autoconf', type='build')
     depends_on('automake', type='build')
     depends_on('libtool', type='build')
+
+    @run_after('install')
+    def cache_test_sources(self):
+        """Copy the example source files after the package is installed to an
+        install test subdirectory for use during `spack test run`."""
+        self.cache_extra_test_sources(['tests', 'include/config.h'])
```

Comment:
Since people are actively working on windows support, using `join_path` is more portable:

```suggestion
        self.cache_extra_test_sources(['tests', join_path('include', 'config.h')])
```